### PR TITLE
Upgrade to CMake 3.11

### PR DESCRIPTION
**Pull request by klecki**
CMake 3.11 was released on March 28th, 2018.

Signed-off-by: Krzysztof Lecki <klecki@nvidia.com>

### CODE REVIEW

**Comment by klecki** [comment]
!build

File: CMakeLists.txt [hunk 1]
```diff
@@ -12,7 +12,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-cmake_minimum_required(VERSION 3.5)
+cmake_minimum_required(VERSION 3.11)
```
Comment (jantonguirao) [inline CMakeLists.txt]: any reason behind this?

Comment (klecki) [reply CMakeLists.txt]: We have some workarounds for older CMakes, that could be cleaned up along the way, and I want to use newer FindCuda that has better support for cross-compiling.

**Comment by JanuszL** [comment]
Please update the readme requirements for DALI build, add information how to update CMake to given version and we can remove FindProtobuf.cmake.

**Comment by klecki** [comment]
> Please update the readme requirements for DALI build, add information how to update CMake to given version and we can remove FindProtobuf.cmake.

Will do, but I intend to leave FindProtobuf.cmake for now, as it does not understand cross-compilation AFAIK.

**Comment by klecki** [comment]
!build

**Pull request merged**